itests/util/src/main/java/org/apache/hadoop/hive/ql/QOutProcessor.java

```diff
@@ -310,6 +290,17 @@ LineProcessingResult processLine(String line) {
     return result;
   }
 
+  private void maskPattern(LineProcessingResult result, Mask mask, PatternReplacementPair patternReplacementPair) {
+    Matcher matcher;
```

Comment:
this declaration of matcher variable can go together with the definition, we have no reason to have the separated